### PR TITLE
free heroku (with upgrade instructions) and allow spoke to function without any additional parallel process

### PR DESCRIPTION
**Pull request by schuyler1d**
… for handling messages, at least for twilio

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,11 +46,6 @@ If you want to create an invite via the home page "Login and get started" link,
 
 See https://github.com/MoveOnOrg/Spoke/blob/master/docs/HOWTO-run_tests.md
 
-## Important TODOS
```
Comment (shakalee14) [inline README.md:52]: glad we're finally deleting this

File: docs/HOWTO_HEROKU_DEPLOY.md [hunk 2]
```diff
@@ -1,24 +1,10 @@
 # Instructions for one click deployment to heroku
-- Create a heroku account (if you don't have an account)- you will need to connect a CC to your account
-- WARNING - this template will incur costs.
+- Create a heroku account (if you don't have an account)- you will need to connect a credit card to your account
 - Fill out environment variables in form --> instructions about that below
-- Currently works when dyno settings for worker processes are set at `standard` --> for more information about dynos, visit [here](https://devcenter.heroku.com/articles/dynos)
-- Currently works when Heroku Postgres add on is set at the `basic` level
+- Unless you change the `JOBS_SAME_PROCESS` environment variable, do not start any of the processes besides `web`
```
Comment (shakalee14) [inline docs/HOWTO_HEROKU_DEPLOY.md]: I think we need to be explicit that processes is the same thing as dynos.

**Review by shakalee14 — APPROVED** [review]

**Comment by shakalee14** [comment]
How can I test this pr - if you press `deploy to heroku` on https://github.com/MoveOnOrg/Spoke/tree/wip-1process-heroku-friendly, it auto deploys the master branch. I know when I was working on this, I had to use my fork.

**Comment by schuyler1d** [comment]
you can visit this url: https://heroku.com/deploy?template=https://github.com/MoveOnOrg/Spoke/tree/wip-1process-heroku-friendly

**Pull request merged**